Makefile

```diff
@@ -1,3 +1,11 @@
+BUILDTAGS ?= \
+	$(shell hack/btrfs_installed_tag.sh) \
+	$(shell hack/btrfs_tag.sh) \
+	exclude_graphdriver_devicemapper \
```

Comment:
what's the purpose of it?

Reply:
RHEL btrfs problems, I'm going to leave commented in case of needed